editor/src/components/canvas/canvas-strategies/strategies/absolute-resize-bounding-box-strategy.tsx

```diff
@@ -289,19 +289,24 @@ export function absoluteResizeBoundingBoxStrategy(
               ]
             })
 
-            return strategyApplicationResult([
-              ...commandsForSelectedElements,
-              setSnappingGuidelines('mid-interaction', guidelinesWithSnappingVector),
-              updateHighlightedViews('mid-interaction', []),
-              setCursorCommand(pickCursorFromEdgePosition(edgePosition)),
-              setElementsToRerenderCommand(retargetedTargets),
-            ])
+            return strategyApplicationResult(
+              [
+                ...commandsForSelectedElements,
+                setSnappingGuidelines('mid-interaction', guidelinesWithSnappingVector),
+                updateHighlightedViews('mid-interaction', []),
+                setCursorCommand(pickCursorFromEdgePosition(edgePosition)),
+              ],
+              'rerender-all-elements',
```

Comment:
this should be retargetedTargets based on the diff

Reply:
Fixed in 71214dfb2.